timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -32,9 +41,88 @@
 from .model_trial import model_trial, skip_hpo
 
 logger = logging.getLogger(__name__)
+dup_filter = DuplicateFilter()
+logger.addFilter(dup_filter)
```

Comment:
Why do we need this?

Reply:
carried over from the original `AbstractModel`. Will probably be gone when I touch the model.